packages/client/src/containers/Dashboard/Dashboard.css

```diff
@@ -0,0 +1,272 @@
+  justify-content: center;
+  align-items: center;
+  gap: 18.204px;
+  flex: 1 0 0;
+  color: var(--primary-disabled-text);
+}
+
+.content-container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  gap: 80px;
+  flex-shrink: 0;
+  flex-grow: 1;
+  height: 100vh;
+  overflow-x: hidden;
+}
+
+.top-nav-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  padding: 40px 80px;
+  background: var(--primary-background-color);
+}
+
+.heading-text {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  flex: 1 0 0;
+}
+
+.sub-heading {
+  color: var(--light-subheading-text-color);
+}
+
+.subtext {
+  color: var(--light-sub-text-color);
+}
+
+.image-container {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.placeholder-image {
+  width: 50px;
+  height: 50px;
+  flex-shrink: 0;
+  border-radius: 50px;
+  border: 3px solid #1d4ed8;
+  background: linear-gradient(
+      0deg,
+      rgba(217, 217, 217, 0.1) 0%,
+      rgba(217, 217, 217, 0.1) 100%
+    ),
+    lightgray 50% / cover no-repeat;
+}
+
+.name-mail-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  flex: 1 0 0;
+}
+
+.team-members-container {
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.teamMembers-heading {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start; /* Align the text to the left */
+  margin-bottom: 20px; /* Add margin for spacing */
+}
+
+.team-members-container ul {
+  list-style-type: none; /* Remove default list styles */
+  padding: 0; /* Remove default padding */
+  background: var(--primary-background-color);
+  padding: 20px 20px 20px 20px;
+}
+
+.team-members-container ul li {
+  border-bottom: 1px solid #ccc; /* Add bottom border */
+  padding: 10px 0; /* Add padding top and bottom */
+}
+
+.team-members-container ul li:last-child {
+  border-bottom: none; /* Remove bottom border for the last item */
+}
+
+.member-info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #ccc;
+  padding: 10px 0;
+}
+
+.team-members-heading-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0px;
+}
+
+.add-member {
+  margin-top: 0px;
+  margin-bottom: 20px;
+}
+.edit-member-container {
+  margin-top: 20px;
+  background-color: #fff;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+}
+
+.edit-member-container h2 {
+  font-size: 20px;
+  margin-bottom: 10px;
+  color: #333;
+}
+
+.edit-member-container input[type='text'] {
+  width: calc(100% - 20px); /* Adjust width to fit container padding */
+  padding: 10px;
+  margin-bottom: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+}
+
+.edit-member-container button {
+  padding: 10px 20px;
+  background-color: #007bff;
```

Comment:
Since you are already using variables, you could add some more repeating values there, especially the colors. 🙏🏽

Reply:
I agree. I did add a var and re wrote a few of the buttons. but if we have more time I will make more var for the other colors elements as well.

Reply:
Sorry, I did not figure this out, I could have. 🙈